uint16 real1
uint16 imaginary1
uint16 real2
uint16 imaginary2
char oper